lib/plugins/aws/lib/validate.js

```diff
@@ -5,8 +5,7 @@ const BbPromise = require('bluebird');
 module.exports = {
   validate() {
     if (!this.serverless.config.servicePath) {
-      throw new this.serverless.classes
-        .Error('This command can only be run inside a service directory');
+      return BbPromise.reject('This command can only be run inside a service directory');
```

Comment:
Technically there's nothing _async_ in this function. Why do we make it _async_? Just to maintain convention across all `validate` functions?